### PR TITLE
privacy-center/consent: Cypress tests for consent verification and update

### PR DESCRIPTION
**Pull request by ssangervasi**
### Code Changes

* Installed and configured cypress
* Added tests for consent flow
* Github workflow for these tests

### Steps to Confirm

* Serve the privacy center for tests:`npm run cy:start` (from privacy-center dir)
* Run cypress: `npm run cy:open`
    * Probably need an npm install to get cypress in that dir.

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* Documentation Updated:
  * [ ] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  * [ ] documentation issue created (tag docs-team to complete issue separately)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [x] Update `CHANGELOG.md`

### Description Of Changes

In order to have confidence in what I make for for https://github.com/ethyca/fides/issues/1395, I realized I needed repeatable test cases. Futzing with cookie changes is annoying even with incognito tabs. So, this gets us to a baseline where we can test what the privacy center stores in the browser, which sets me up to write the code that _reads_ the values without privacy center's help.

### CODE REVIEW

File: clients/privacy-center/pages/consent.tsx [hunk 1]
```diff
@@ -44,6 +44,7 @@ const Consent: NextPage = () => {
     const getUserConsents = async () => {
       if (!consentRequestId) {
         router.push("/");
+        return;
```
Comment (LKCSmith) [inline clients/privacy-center/pages/consent.tsx:47]: Why are these returns needed? 🤔

**Review by LKCSmith — APPROVED** [review]
Just a question

File: clients/privacy-center/pages/consent.tsx (hunk 1 above)
Comment (ssangervasi) [reply clients/privacy-center/pages/consent.tsx:47]: Previously the function would keep running even though the logic would all be unnecessary. I noticed the `fetch` on line 87 would fire even though the `verificationCode` was already known to be invalid.

**Review by chriscalhoun1974 — APPROVED** [review]
lgtm. Great work!

**Pull request merged**